.storybook/themes/news.scss

```diff
@@ -633,6 +633,46 @@
 				right-rail-advanced-main-right-rail-bottom: (
 					grid-area: main-right-rail-bottom,
 				),
+				share-bar: (
+					box-shadow: (
```

Comment:
looks like in the mocks in figma it's using a box-shadow-1 alias 

		"box-shadow-1": 0px 0px 16px rgba(25, 25, 25, 0.4),

![Screen Shot 2022-06-02 at 15 55 56](https://user-images.githubusercontent.com/5950956/171736681-54d4d330-b75d-444a-9c24-3396fece0270.png)

Reply:
@JackHowa : Where can I find these specifications in Figma? Thanks!

Reply:
the variable is also available in the style sheet

Reply:
@JackHowa : Thanks! All caught up & using all of these tokens now.